### PR TITLE
script_output: increase logging in case of failure

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by Martchus** [comment]
Looks good, beside that it isn't formatted with `tools/tidy`.

File: distribution.pm [hunk 1]
```diff
@@ -245,10 +245,16 @@ sub script_output {
         testapi::type_string("($run_script) | tee /dev/$testapi::serialdev\n");
     }
     my $output = testapi::wait_serial("SCRIPT_FINISHED$marker-\\d+-", $wait, 0, record_output => 1)
-      || croak "script timeout";
+      || croak "script timeout: $script";
 
     if ($output !~ "SCRIPT_FINISHED$marker-0-") {
-        croak "script failed with : $output" unless $args{proceed_on_failure};
+        my $log_message = 'script failed with :' . $output;
```
Comment (Martchus) [inline distribution.pm]: The space should go after the colon.

**Comment by Martchus** [comment]
I would ignore the coverage checks (eg. the check for patch shows lots of red lines which aren't actually part of the patch).

File: distribution.pm (hunk 1 above)
Comment (asmorodskyi) [reply distribution.pm]: Done

**Comment by okurz** [comment]
Could you provide a link to an openQA test which shows the difference in action?

**Comment by asmorodskyi** [comment]
http://kimball.arch.suse.de/tests/1761/file/autoinst-log.txt - here you go , look for "distribution::script_output(script failed with :"

**Comment by okurz** [comment]
+1

**Pull request merged**